src/resources/Customer.js

```diff
@@ -329,6 +350,26 @@ class Customer extends Resource {
     return this.resource(Trip, Trip.makeHref(this.code, id));
   }
 
+  /**
+   * Gets a user resource by id
+   * @param {Object} id Identity of the user or the new object
+   * @returns {User} User resource
+   */
+  user(id) {
+    if (!isNaN(parseFloat(id)) && isFinite(id)) {
+      return this.resource(User, User.makeHref(this.code, id));
+    }
+    return this.resource(User, { code: this.code, ...id });
+  }
+
+  /**
+   * Gets a context for querying this customer's users
+   * @returns {UserContext} Context for querying this customer's users
+   */
+  users() {
```

Comment:
This still make sense here as long as the express intention is to get users associated to this customer.

Reply:
Yeah, given the context this will be used in that is the intention.  I'll update the jsdoc slightly to make that more explicit.  Thanks!